### PR TITLE
Create new layout for own publications

### PR DESCRIPTION
**Pull request by jankrcal**
This commit adds a new layout for publications of Fakta o klimatu.

### CODE REVIEW

**Comment by jankrcal** [comment]
@mgrabovsky Could you PTAL or should I find someone else? (we want to publish this on next Tuesday)

**Comment by mgrabovsky** [comment]
I'll take a look later this week.

File: _layouts/publication.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  {% include navigation.html %}
+  <main>
+    <div class="section publication-cover" style="background-color: {{ page.background_color }}">
+      {% assign download_path = "/assets/studies/" | append: page.slug %}
+      <div class="container">
+        <div class="titles">
+          <div class="publication-type">
+            {{ page.type | default: site.data.lang.text.publication.publication }}
+          </div>
+          <h1>{{ page.title_br | default: page.title }}</h1>
```
Comment (mgrabovsky) [inline _layouts/publication.html]: I'd prefer a more descriptive name for the attribute, perhaps `page.title_formatted` or `page.title_html`?

File: _layouts/publication.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <div class="subtitle">{{ page.subtitle }}</div>
+        </div>
+        <div class="cover-img" style="background-image: url('{{ download_path }}.svg')"></div>
+        <div class="download">
+          <a href="/assets-local/publications/{{page.slug }}.pdf" class="btn btn-outline-light"><i class="fas fa-fw fa-download"></i>
```
Comment (mgrabovsky) [inline _layouts/publication.html]:
```diff
-<a href="/assets-local/publications/{{page.slug }}.pdf" class="btn btn-outline-light"><i class="fas fa-fw fa-download"></i>
+<a href="/assets-local/publications/{{ page.slug }}.pdf" class="btn btn-outline-light"><i class="fas fa-fw fa-download"></i>
```

File: _layouts/publication.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            {{ page.download | default: site.data.lang.text.publication.download }}
+          </a>
+        </div>
+        {%- if page.sponsor %}
+          <div class="sponsor">
+            <span class="title">{{ site.data.lang.text.publication.sponsor}}:</span>
```
Comment (mgrabovsky) [inline _layouts/publication.html]:
```diff
-<span class="title">{{ site.data.lang.text.publication.sponsor}}:</span>
+<span class="title">{{ site.data.lang.text.publication.sponsor }}:</span>
```

File: _layouts/publication.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            {%- if page.sponsor_link %}
+              <a href="{{ page.sponsor_link}}" class="content">{{ page.sponsor }}</a>
```
Comment (mgrabovsky) [inline _layouts/publication.html]:
```diff
-<a href="{{ page.sponsor_link}}" class="content">{{ page.sponsor }}</a>
+<a href="{{ page.sponsor_link }}" class="content">{{ page.sponsor }}</a>
```

File: assets/_scss/publication.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
+/* Publication page customizations */
```
Comment (mgrabovsky) [inline assets/_scss/publication.scss:1]: Note: We use 4 spaces for indentation.

File: assets/_scss/publication.scss [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    }
+  }
+}
+
+.download {
+  margin-block: 2rem;
+  a { font-size: 1.4rem; }
+}
+
+.longread {
```
Comment (mgrabovsky) [inline assets/_scss/publication.scss]: Do we want this styling (especially the headings and arrow lists) for all our publication pages? It seems a bit too opinionated to my liking.

**Review by mgrabovsky — COMMENTED** [review]
Couple of comments. Looks fine overall.

File: assets/_scss/tags.scss [hunk 7]
```diff
@@ -4,10 +4,12 @@
     top: auto;
     padding: 40px 15px 15px 15px;
     overflow: hidden;
-    background-image: linear-gradient(rgba(255,255,255,0), white 80%);
 
     .tags {
         width: max-content;
+        .btn-tag {
+            box-shadow: 0 0px 5px white;
```
[the remark this answers is not in the document]
Comment (jankrcal) [reply assets/_scss/tags.scss:11]: Done.

File: assets/_scss/publication.scss (hunk 5 above)
Comment (jankrcal) [reply assets/_scss/publication.scss:1]: Fixed.

File: _layouts/publication.html (hunk 4 above)
Comment (jankrcal) [reply _layouts/publication.html]: Done.

File: _layouts/publication.html (hunk 3 above)
Comment (jankrcal) [reply _layouts/publication.html]: Done.

File: _layouts/publication.html (hunk 2 above)
Comment (jankrcal) [reply _layouts/publication.html]: Done.

File: _layouts/publication.html (hunk 1 above)
Comment (jankrcal) [reply _layouts/publication.html]: Yep, `title_formatted` sgtm.

File: _layouts/publication.html [hunk 8]
```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html lang="{{ site.lang }}">
+<head>
+  {% include header.html %}
+</head>
+
+<body>
+  {% include navigation.html %}
+  <main>
+    <div class="section publication-cover" style="background-color: {{ page.background_color }}">
```
[the remark this answers is not in the document]
Comment (jankrcal) [reply _layouts/publication.html]: Added 'dimgray' as default (just to make sure the white text is readable on top of that).

File: assets/_scss/publication.scss (hunk 6 above)
Comment (jankrcal) [reply assets/_scss/publication.scss]: This is an outdated version of the file: in the latest one, I have this guarded by a document class.

**Comment by jankrcal** [comment]
Thanks! Could you please take another look?

**Review by mgrabovsky — APPROVED** [review]
Looks good. Thanks for the improvements.

**Pull request merged**